src/commands/utils/azurestorage.ts

```diff
@@ -9,7 +9,7 @@ export enum LinkDuration {
 function sasDuration(duration: LinkDuration): number {
     switch (duration) {
         case LinkDuration.StartTime: return 2 * 60 * 1000;
-        case LinkDuration.DownloadNow: return 5 * 60 * 1000;
+        case LinkDuration.DownloadNow: return 15 * 60 * 1000; // 15 minutes to allow for Windows image pulls and log export
```

Comment:
💡Note: This will not change any behaviour in existing cases because its just to handle windows. Please correct me if I am wrong. 🥷